java/plasma/src/main/java/org/apache/arrow/plasma/PlasmaClient.java

```diff
@@ -105,6 +106,11 @@ public void put(byte[] objectId, byte[] value, byte[] metadata)
     return ret;
   }
 
+  @Override
+  public List<byte[]> list() {
```

Comment:
This seems to throw away metadata from List, why?

Reply:
Yes, list just return objectId in plasma store. You can get metadata by call get(), list() will not return metadata.

Reply:
get lookslike it requires getting the actual objects though?  Given the parameters on get() would itbe better to return byte[][] instead?

Reply:
I think I have misunderstand about your metadata. get could return object's data and metadata(user decide, could be null, not necessary).
If you mean some info in ObjectTableEntry such as mapped file fd, offset, I think it's not needed for Java code.